sympy/core/tests/test_equation.py

```diff
@@ -0,0 +1,72 @@
+from sympy import var, integrate, simplify, log, Integral
+from ..relational import Equality
+from ..function import *
+from ..equation import *
+from pytest import raises
+
+def test_define_equation():
+    a,b,c=var('a b c')
+    raises(NotImplementedError, lambda: Equation(a,b/c,'>'))
+    tsteqn = Eqn(a,b/c)
+    assert Equation(a,b/c) == Equation(a,b/c,'=')
+    assert tsteqn == Equation(a,b/c, '=')
+    assert tsteqn.lhs == Equation(a,b/c,'=').args[0]
+    assert tsteqn.rhs == Equation(a,b/c,'=').args[1]
+    assert tsteqn.relop == Equation(a,b/c,'=').args[2]
+    assert tsteqn.free_symbols == {a, b, c}
+    
+def test_convert_equation():
+    a,b,c=var('a b c')
+    tsteqn = Eqn(a,b/c)
+    assert tsteqn.as_Boolean() == Equality(a,b/c)
+    assert tsteqn.reversed == Equation(b/c,a,'=')
+    
+def test_binary_op():
+    a,b,c=var('a b c')
+    tsteqn = Eqn(a,b/c)
+    assert tsteqn.__add__(c) == Equation(a + c, b/c + c)
```

Comment:
I think it's better to test e.g. `a + b` and `b + a` rather than `a.__add__(b)`

Reply:
good point. will fix

Reply:
Done in updated tests.